### ISSUE TITLE
[1.52.2] 'gradlew runClient' fails

### ISSUE THREAD

**Issue by Spongman** [FTBTeam/FTB-Quests#490]
````
> gradlew.bat runClient
...
[16:48:31] [main/DEBUG] [mixin/]: Mixing LivingEntityMixin from kubejs.mixins.json into net.minecraft.entity.LivingEntity
[16:48:32] [main/FATAL] [mixin/]: Mixin apply failed kubejs.mixins.json:LivingEntityMixin -> net.minecraft.entity.LivingEntity: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Critical injection failure: @Inject annotation on foodEaten could not find any targets matching 'Lnet/minecraft/entity/LivingEntity;func_213357_a(Lnet/minecraft/world/World;Lnet/minecraft/item/ItemStack;)Lnet/minecraft/item/ItemStack;' in net.minecraft.entity.LivingEntity. Using refmap kubejs.refmap.json [PREINJECT Applicator Phase -> kubejs.mixins.json:LivingEntityMixin -> Prepare Injections ->  -> handler$zza000$foodEaten(Lnet/minecraft/world/World;Lnet/minecraft/item/ItemStack;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;)V -> Parse]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: @Inject annotation on foodEaten could not find any targets matching 'Lnet/minecraft/entity/LivingEntity;func_213357_a(Lnet/minecraft/world/World;Lnet/minecraft/item/ItemStack;)Lnet/minecraft/item/ItemStack;' in net.minecraft.entity.LivingEntity. Using refmap kubejs.refmap.json [PREINJECT Applicator Phase -> kubejs.mixins.json:LivingEntityMixin -> Prepare Injections ->  -> handler$zza000$foodEaten(Lnet/minecraft/world/World;Lnet/minecraft/item/ItemStack;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;)V -> Parse]
````

**Comment by Spongman** [comment]
wrong java version.

**Comment by Spongman** [comment]
@LatvianModder can you please ensure that the 1.15.2 branch builds?

**Comment by MaxNeedsSnacks** [comment]
This all ties into an old mixin issue with refmaps on Forge, so you'd have to fix that on KubeJS' end or downgrade the KubeJS version used in the dev env. As for 1.15 support on either of these mods, we have discontinued both Quests and KubeJS for that version, and now unfortunately the code bases are too different to backport a lot of meaningful changes.

**Comment by Spongman** [comment]
that's a real shame, since it seems like most modpacks released today are still on 1.15...
